docs/s3cmd-with-minio.md

```diff
@@ -1,105 +1,176 @@
 
-## 3. Configuration
+## <a name="installs3cmd"></a>2. Install `S3cmd`
 
-We will run `S3cmd` on <https://play.minio.io:9000>.
+Install `S3cmd` using these instructions: <http://s3tools.org/s3cmd>.
 
-Access credentials shown in this example belong to <https://play.minio.io:9000>. These credentials are open to public. Feel free to use this service for testing and development. Replace with your own Minio keys in deployment.
+## <a name="configures3cmd"></a>3. Configure `S3cmd`
 
-Edit the following fields in your s3cmd configuration file `~/.s3cfg`
+### 3.1 Generate a Configuration file
+`S3cmd` uses a configuration file called **.s3cfg** to access cloud storage. Use the following command to generate the initial version of **.s3cfg**:
 
 ```sh
-# Setup endpoint
+ `./S3cmd --configure`
+```
+
+**Note:** Accept the defaults when prompted.
+
+### 3.2 Edit the Configuration File
+Modify the configuration file to enable `S3cmd` to manage buckets on https://play.minio.io:9000:
+
+#### 3.2.1. Navigate to **/users/<your user name>** and open **.s3cfg** in a text editor.
+#### 3.2.2. Edit the following fields in **.s3cfg** to configure the endpoint:
+
+```sh
+bucket_location = us-east-1
 host_base = play.minio.io:9000
 host_bucket = play.minio.io:9000
-bucket_location = us-east-1
 use_https = True
+```
 
-# Setup access keys
+#### 3.3.3. Edit the following fields in **.s3cfg** to set the access keys:
+
+```sh
 access_key =  Q3AM3UQ867SPQQA43P2F
 secret_key = zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
+```
+
+#### 3.3.4. Edit the following fields in **.s3cfg** to enable S3 v4 signature APIs:
 
-# Enable S3 v4 signature APIs
+```sh
 signature_v2 = False
 ```
 
-## 4. Commands
+#### 3.3.5. Save **.s3cfg**.
 
-### To make a bucket
+**Note:** The variables set in this example are for public testing and development on <https://play.minio.io:9000>. Modify these variables as appropriate when developing for your own Minio Server.
+
+
+## <a name="runs3cmdcommands"></a>4. Examples of Typical `S3cmd` Commands
+Navigate to the installation directory for **S3cmd** and invoke the following commands to create and manage a bucket:
+
+### Create a Bucket
 
 ```sh
 s3cmd mb s3://mybucket
+```
+
+You should see a response similar to this one:
+
+```sh
 Bucket 's3://mybucket/' created
 ```
 
-### To copy an object to bucket
+### Copy an Object to the Bucket
 
 ```sh
 s3cmd put newfile s3://testbucket
+```
+
+You should see a response similar to this one:
+
+```sh
 upload: 'newfile' -> 's3://testbucket/newfile'  
 ```
 
-### To copy an object to local system
+### Copy an Object to the Local File System
 
 ```sh
 s3cmd get s3://testbucket/newfile
+```
+
+You should see a response similar to this one:
+
+```sh
 download: 's3://testbucket/newfile' -> './newfile'
 ```
 
-### To sync local file/directory to a bucket
+### Sync a Local File/Directory to a Bucket
```

Comment:
i think we can just say "Sync a local file/directory to a bucket"
Remove unnecessary capitalization.

Reply:
@deekoder Disagree for same reasons as above.